### PR TITLE
Update kubevirt artifact's format

### PR DESCRIPTION
**Pull request by gursewak1997**
Updating the format for kubevirt artifacts in `rationale.yaml` and `sample.json` for streams and releases metadata.

### CODE REVIEW

File: metadata/release/sample.json [hunk 1]
```diff
@@ -104,10 +104,10 @@
         },
         "kubevirt": {
           "artifacts": {
-            "qcow2.xz": {
+            "ociarchive": {
               "disk": {
-                "location": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20190801.0/x86_64/fedora-coreos-30.20190801.0-kubevirt.qcow2.xz",
-                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20190801.0/x86_64/fedora-coreos-30.20190801.0-kubevirt.qcow2.xz.sig",
+                "location": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20190801.0/x86_64/fedora-coreos-30.20190801.0-kubevirt.ociarchive",
```
Comment (bgilbert) [inline metadata/release/sample.json:109]: I tried `cosa buildextend-kubevirt` followed by `cosa compress` and it created an `ociarchive.xz`.  If that is correct, we should append `.xz` to `ociarchive` in all the kubevirt sections here.

Comment (dustymabe) [reply metadata/release/sample.json:109]: I'm thinking we don't want an `ociarchive.xz` but rather a `.ociarchive` like we do for our other container artifacts we create. We'll probably need to update `cosa` to do the right thing here.

Comment (dustymabe) [reply metadata/release/sample.json:109]: Thanks for catching this @bgilbert

Comment (bgilbert) [reply metadata/release/sample.json:109]: Okay, in that case we'll want to drop the `uncompressed-sha256` here.

**Review by bgilbert — APPROVED** [review]
LGTM

**Pull request merged**